src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Extensions/DocumentHelperExtensions.cs

```diff
@@ -12,6 +12,16 @@
 
 using Newtonsoft.Json.Serialization;
 
+#if NETSTANDARD2_0
+using FA = Microsoft.Azure.WebJobs.FunctionNameAttribute;
+using TA = Microsoft.Azure.WebJobs.HttpTriggerAttribute;
+#endif
+
+#if NET5_0

```

Comment:
This was my first approach. But, it might become the issue when .NET 6, .NET 7, etc keep coming.